frontend/src/container/LogsExplorerViews/index.tsx

```diff
@@ -334,9 +340,11 @@ function LogsExplorerViews(): JSX.Element {
 
 					const dashboardEditView = `${generatePath(ROUTES.DASHBOARD, {
 						dashboardId: data?.payload?.uuid,
-					})}/new?${QueryParams.graphType}=graph&${QueryParams.widgetId}=empty&${
-						queryParamNamesMap.compositeQuery
-					}=${encodeURIComponent(JSON.stringify(exportDefaultQuery))}`;
+					})}/new?${QueryParams.graphType}=${panelTypeParam}&${
```

Comment:
can we have common handler though in future we would adding another explorer page which would be good to handle from one single source of truth